README.md

```diff
@@ -83,5 +83,41 @@ Credit: [@omundy](https://github.com/omundy)
 With `InventoryObject` selected, in the Canvas Scaler component, set `Reference Pixels Per Unit` to 32.
 
 
+#### Chapter 7, Page 263
+Credit: [@omundy](https://github.com/omundy)
+
+In the **The DamageCharacter() method** section, the code should be added to the `Enemy` class.
+
+
+#### Chapter 7, Page 269
+Credit: [@omundy](https://github.com/omundy)
+
+In the **Refactoring Prefab Instantiation** section, remove all *four* lines from `Start()` and add them to the new `ResetCharacter()` override.
+
+After you add the `ResetCharacter()` method to the `Player` class, add the following method
+
+```
+private void OnEnable() {
+	ResetCharacter();
+}
+```
+
+#### Chapter 7, Page 275
+Credit: [@omundy](https://github.com/omundy)
+
+You'll notice when you run into the enemies they spin. Like you did with the PlayerObject, check `Freeze Rotation` for the Z axis inside the Rigidbody2D on the EnemyObject prefab.
```

Comment:
Oomph - good catch!

Reply:
*Actually kind of funny to see them scatter like bowling pins. I think it's good when a reader is allowed to make small errors and have a moment to think about what the problem may be *before* the text guides them.

Reply:
Agreed. Personally, I learn best by breaking things then taking a moment to think about it. The first time the player runs into a collider (page 134), the `Freeze Rotation Z` box isn't checked, and they can see the weird results.

If I do a 2nd edition of this book, perhaps I'll make a point of instructing the reader to pause and consider *why* something is broken.